deploy/sre-prometheus/management-cluster/100-dynatrace-workloads-monitoring.PrometheusRule.yaml

```diff
@@ -0,0 +1,91 @@
+      for: 10m
+      labels:
+        namespace: dynatrace
+        severity: critical
+      annotations:
+        summary: "The Dynatrace Operator is Degraded"
+        description: "{{$value}} pods are not running out of total pods for the Dynatrace Operator"
+  
+    - alert: DynatraceWebhookDegradedSRE
+      expr: (sum(kube_deployment_spec_replicas{deployment="dynatrace-webhook", namespace="dynatrace"}) without (deployment, instance, pod)) - (sum(kube_deployment_status_replicas_available{deployment="dynatrace-webhook", namespace="dynatrace"}) without (deployment, instance, pod)) > 0
+      for: 10m
+      labels:
+        namespace: dynatrace
+        severity: critical
+      annotations:
+        summary: "Dynatrace Webhook is Degraded"
+        description: "{{$value}} pods are not running out of total pods for the Dynatrace Webhook"
+
+    - alert: DynatraceOpenTelemetryCollectorDegradedSRE
+      expr: (sum(kube_deployment_spec_replicas{deployment="opentelemetry-dynatrace-collector", namespace="dynatrace"}) without (deployment, instance, pod)) - (sum(kube_deployment_status_replicas_available{deployment="opentelemetry-dynatrace-collector", namespace="dynatrace"}) without (deployment, instance, pod)) == 0
+      for: 15m
+      labels:
+        namespace: dynatrace
+        severity: critical
+      annotations:
+        summary: "Dynatrace OTEL Collector is Degraded"
+        description: "{{$value}} pods are not running out of total pods for the OpenTelemetry Dynatrace Collector"
+
+    - alert: DynatraceActiveGateDegradedSRE
+      expr: (sum(kube_statefulset_status_replicas{statefulset=~".*-activegate", namespace="dynatrace"}) without (deployment, instance, pod)) - (sum(kube_statefulset_status_replicas_ready{statefulset=~".*-activegate", namespace="dynatrace"}) without (deployment, instance, pod)) > 0
+      for: 15m
+      labels:
+        namespace: dynatrace
+        severity: critical
+      annotations:
+        summary: "Dynatrace ActiveGate is Degraded"
+        description: "{{$value}} pods are not running out of total pods for Dynatrace ActiveGate"
+
+    - alert: DynatraceOneAgentDegraded
+      expr: sum(kube_daemonset_status_desired_number_scheduled{daemonset=~".*-oneagent", namespace="dynatrace"}) without (daemonset, instance, pod) - sum(kube_daemonset_status_number_ready{daemonset=~".*-oneagent", namespace="dynatrace"}) without (daemonset, instance, pod) > 0
+      for: 15m
+      labels:
+        namespace: dynatrace
+        severity: critical
+      annotations:
+        summary: "Dynatrace OneAgent is Degraded"
+        description: "{{$value}} pods are not running out of total pods for Dynatrace OneAgent"  
+
+    - alert: DynatracePodRestartsStuck
```

Comment:
Why does this alert not contain the SRE suffix while others do?

Reply:
Thanks Manuel, we'll fix this.

Reply:
Added the SRE Suffix